api/resources/translations/messages-en.properties

```diff
@@ -1272,6 +1280,7 @@ translation.add = Add new translation key
 translation.key = Translation key
 unique.id = Unique ID
 unknown.contact = Unknown contact
+update.password.help = User will be required to reset their password on their next login
```

Comment:
Hey folks could I please get a review of this translation. Feel free to add the translation in this comment. 

French - @andrablaj 
Nepali - @sugat009 
Spanish - @lorerod 

Thank you.

Reply:
French: `L'utilisateur devra réinitialiser son mot de passe lors de sa prochaine connexion`

Reply:
Spanish: El usuario deberá restablecer su contraseña en su próximo inicio de sesión.